### ISSUE TITLE
[FEATURE] Fix the broken transit shape by applying transit routing changes

### ISSUE THREAD

**Issue by yueshuaing** [wsp-sag/network_wrangler#390]
### User Story
In Metcouncil future year workflows, some roadway project cards remove roadways that have transit lines running on them. Currently, NW stops if any roadway changes break the existing transit routes. We'd like to have the feature that allows using the transit routing changes to fix the route after applying the roadway deletion. 

### To Reproduce
Run `pytest tests\test_scenario.py`
The function to test is `test_scenario_apply_highway_and_transit_changes`

**Comment by e-lo** [comment]
> NW stops if any roadway changes break the existing transit routes.

Yeah, this isn't actually a good default behavior and I'd consider it a bug.  Rather, we should present a warning.
Consider a roadway change that splits a link and a transit change that fixes it or reroutes around the split link.  They are dependent on each other and thus should be applied as *co-requisites* or even within the same project file.  That said, when the change is made to transit the resulting highway network is not updated with the new link yet.  Or if we change transit first, it will try and route it on a link that doesn't yet exist creating a chicken-and-egg issue which can only be solved by splitting up the changes into 1. add new link. 2. reroute transit 2. delete old link.  This sucks as a user experience. 

Therefore, I'm changing the default behavior to a **warning** which should require the user to review what was going on.
